lua/tint.lua

```diff
@@ -414,6 +414,16 @@ __.setup_all = function(skip_config)
 
   setup_namespaces()
   setup_autocmds()
+
+  on_or_after_vimenter(vim.schedule_wrap(function()
```

Comment:
Why does this need to be `on_or_after_vimenter`? The calls to `tint.setup` already ensure that the main setup happens in this manner, right?

Reply:
You're right. I should actually use vim.schedule, because vim.schedule_wrap returns a function, it doesn't call it, and that's the problem I was having.